example/lib/main.dart

```diff
@@ -14,29 +15,24 @@ class MyApp extends StatefulWidget {
 }
 
 class _MyAppState extends State<MyApp> {
-  String _platformVersion = 'Unknown';
+  String _platformVersion = ExampleStrings.platformVersion;
 
   @override
   void initState() {
     super.initState();
-    initPlatformState();
+    //initPlatformState();
   }
 
-  // Platform messages are asynchronous, so we initialize in an async method.
   Future<void> initPlatformState() async {
```

Comment:
if you are not going to use the method to show the platform version, please remove the implementation.
And if you opt for the removal, do it on the whole plugin, not just on this file

Reply:
This method was removed from whole project finally.